src/launch/config.c

```diff
@@ -1109,12 +1136,26 @@ static void config_parser_end_fn(void *userdata, const XML_Char *name) {
                 break;
         }
 
+        case CONFIG_NODE_LIMIT: {
+                unsigned long long limit;
+                char *end;
+
+                errno = 0;
+                limit = strtoull(state->current->cdata, &end, 10);
+                if (end != state->current->cdata && *end == '\0' && errno == 0)
```

Comment:
Errrr, this hard-codes `sizeof(long long)` in user-visible stuff. This is 64bit everywhere, I know, so probably fine. But... eh... if "long long" is not 64bit, this suddenly has different maximums depending on the machine. Not sure that's what we want.

Can we make this a util/ helper? util_strtou64()? and then "static_assert(...)"?

Reply:
Will do.